lib/KLabeledIcon.vue

```diff
@@ -62,6 +62,13 @@
         default: '100%',
       },
     },
+    data() {
```

Comment:
Could you just add this to the `styles` object in `labelStyle`? It doesn't need to be reactive.